tests/csrf_tests/tests.py

```diff
@@ -295,6 +347,35 @@ def test_token_node_with_new_csrf_cookie(self):
         csrf_cookie = resp2.cookies[settings.CSRF_COOKIE_NAME]
         self._check_token_present(resp, csrf_id=csrf_cookie.value)
 
+    def test_cookie_not_reset_on_accepted_request(self):
+        """
+        The csrf token used in posts is changed on every request (although stays equivalent).
```

Comment:
I don't think flake8 does any checking of docstrings. The 79 char suggested limit is mentioned in https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style.